packages/p2p/src/components/my-ads/my-ads.scss

```diff
@@ -684,6 +684,13 @@
             padding-right: 1.6rem;
         }
     }
+
+    &__expand-button {
+        background-color: unset;
```

Comment:
can we pass `transparent` to `Button` instead?

Reply:
Ok. I will use that prop